mix.exs

```diff
@@ -40,8 +40,11 @@ defmodule Membrane.AudioMix.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.10.0"},
-      {:membrane_common_c, "~> 0.12.0"},
+      {:membrane_core, "~> 0.10.0", override: true},
+      {:membrane_common_c,
+       git: "https://github.com/membraneframework/membrane_common_c.git",
+       branch: "raw-audio",
+       override: true},
```

Comment:
Obviously, this PR is blocked until https://github.com/membraneframework/membrane_common_c/pull/32 is merged